.circleci/config.yml

```diff
@@ -467,7 +528,39 @@ workflows:
             branches:
               only: main
           requires:
-            - push-qkm-dev-image-quorum-key-manager:develop
+            - push-qkm-dev-image-${CIRCLE_SHA1:0:7}
+      # Will run the end 2 end tests locally for each `/^v.*/` git tag and every main commit
+      # requires successful Build, Unit and acceptance tests
+      - e2e:
+          test_data: $QA_TEST_DATA
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
+              only: main
+          requires:
+            - deploy-qa
+            # Will build and push a dev image for each main commit
+            # Resulting image is tagged with develop
+            # Images are compliant with linux/amd64 architecture
+      - build-docker-image: # Push Dev image
+          name: push-qkm-dev-image-develop
+          matrix:
+            parameters:
+              docker-image-tag: [ "quorum-key-manager:develop" ]
```

Comment:
since there is a single argument, there is not need for using matrix here you can pass it ` docker-image-tag` as a regular param

Reply:
Well it used to be a matrix, so I kept it.
Will do the change